cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -6,32 +6,48 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable improvements in `{odo-title}`{nbsp}{odo-ver}
+== Notable changes and improvements in `{odo-title}`{nbsp}{odo-ver}
+
+* A `--now` flag is added for `odo create`, `odo url` and `odo config`. 
++
+Run `odo url create --now` to create the URL in the configuration and propagate all the changes to the cluster
+* An `odo debug info` command is added. The command displays the information whether the debug mode is enabled for the component, whether the port-forward process is running, and information about ports used.
+* `odo push` output now shows the exact error from the OpenShift cluster on failure.
++
+* A `--secure` flag is enabled for `odo url`. It indicates whether a URL is secure. 
+* `odo storage list` now displays the information about the storage state: `Pushed`, `NotPushed` or `Locally Deleted`.
+* `odo debug port-forward` now selects a port automatically if the default port (5858) is occupied.
+* The `--all` flag is renamed to  `--all-apps`.
+* The default PVC size has been increased to 10GiB
+* The JSON output for `odo storage list -o json` has been restructured.
```

Comment:
Does the user need more information about the new structure?

Reply:
Not really. It's mostly a cosmetic change to make it more readable.